spinta/types/store.py

```diff
@@ -91,7 +91,11 @@ def push(context: Context, store: Store, stream: types.GeneratorType):
         model = get_model_by_name(manifest, model_name)
         client_id = client_supplied_ids.replace(model_name, data)
         check(context, model, model.backend, data)
-        inserted_id = push(context, model, model.backend, data)
+        if 'id' in data:
```

Comment:
Won't this clash with `authorize()`, which check's for `set_meta_fields`: https://github.com/atviriduomenys/spinta/pull/17/files#diff-ffd8ae1d2db7b6de5a2ce27e14be4b62R357